charts/fluid/fluid/templates/role/alluxio/rbac.yaml

```diff
@@ -104,13 +110,25 @@ rules:
       - daemonsets/status
       - statefulsets/status
     verbs:
-      - '*'
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
   - apiGroups:
       - coordination.k8s.io
     resources:
       - leases
     verbs:
```

Comment:
I just found that we don't need ClusterRole-level rbac verbs for leases, Role-level is okay because all the leases is created and maintained in a specific namespace. (See [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/cronjob-tutorial/testdata/project/config/rbac/leader_election_role.yaml#L26-L37) for an example). For a more restricted case, maybe we can define the lease's [`resourceName`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources) according to the controller. For example, AlluxioRuntime controller's lease name is fixed to be [`alluxio.data.fluid.io`](https://github.com/fluid-cloudnative/fluid/blob/master/cmd/alluxio/app/alluxio.go#L109).

Reply:
> I just found that we don't need ClusterRole-level rbac verbs for leases, Role-level is okay because all the leases is created and maintained in a specific namespace. (See [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/cronjob-tutorial/testdata/project/config/rbac/leader_election_role.yaml#L26-L37) for an example). For a more restricted case, maybe we can define the lease's [`resourceName`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources) according to the controller. For example, AlluxioRuntime controller's lease name is fixed to be [`alluxio.data.fluid.io`](https://github.com/fluid-cloudnative/fluid/blob/master/cmd/alluxio/app/alluxio.go#L109).

Ok, I will fix it by adding `Role` and `RoleBinding` for leases.